QUARKUS-2489.md

```diff
@@ -0,0 +1,53 @@
+# QUARKUS-2489 OIDC Back channel logout
+
+Jira: https://issues.redhat.com/browse/QUARKUS-2489
+
+This feature is talking about logout flow on an OIDC provider as Keycloak
+
+## Scope of the testing
+
+Backchannel logout applies to `quarkus-oidc` and basically is a standard way to do logout in to a single sign-on service.
```

Comment:
logout from a single sign-on service?

Reply:
I don't agree. Huge portion of SPAs are going to logout directly (call SSO). TBH all Angular apps I saw do that, but I'm sure there are people who don't do that. Therefore what we need to do is test that we actually get callback from Keycloak when logout happen and Quarkus log us out.

Reply:
Sentence is not wrong though, maybe I concentrated on word "standard" to much.

Reply:
What I mean is this, shouldn't be the sentence changed this way?

"standard way to do logout in to a single sign-on service"
 =>
"standard way to do logout out from a single sign-on service"

Reply:
Yes, the main goal of the logout endpoint is to remove all the active sessions in all devices for a given user (except that the user is using an offline token). 
Ref: https://github.com/quarkusio/quarkus/pull/24611/files#diff-75b29fe10300c4376f22f9584de09fe4d6d993f31fb218d3966ccb9d30c8608bR474

Reply:
> maybe I concentrated on wo

Standard ref that Quarkus must implement: https://openid.net/specs/openid-connect-backchannel-1_0.html#Backchannel

Reply:
@pjgg you are right that these constraints need to be implemented by Quarkus (RP), but in order to be sure that Quarkus enforce all of these constraints you would have to call Quarkus for each of these scenarios (e.g. add nonce claim) and check response code as that's your only indicator whether Quarkus validated it. You can't do it in  bulk. These seems a lot like unit tests rather than integration tests. Do we test in such details all other features? If I'm wrong, please just explain it to me. AFAIK I didn't see it in TS.